README.md

```diff
@@ -190,152 +172,83 @@ For a community-curated lists of awesome projects using IPFS, check out [awesome
 - [i18n](https://github.com/ipfs/i18n) - The IPFS Translation Project: crowdsourcing translations of IPFS GUIs and websites.
-- [pm-ipfs-gui](https://github.com/ipfs-shipyard/pm-ipfs-gui) - Coordinating development and maintenance of GUI apps.
-
-#### Apps on IPFS
-
-You can find an ever growing list at [Awesome IPFS](https://awesome.ipfs.io/apps/)
-
-#### Community Infrastructure
-
-- [blog](https://github.com/ipfs/blog) - The IPFS Blog.
-- [community forum](https://discuss.ipfs.io)
-- [distributions](https://github.com/ipfs/distributions) - Scripts to build the /install html page.
-- [newsletter](https://github.com/ipfs/newsletter) - Prepare and store IPFS Newsletter roundups. [Subscribe here.](http://eepurl.com/gL2Pi5)
-- [infrastructure](https://github.com/ipfs/infrastructure) - Tools and systems for the community.
-- [ops-requests](https://github.com/ipfs/ops-requests) - Requests about infrastructure operations.
-- [project-repos](https://github.com/ipfs/project-repos) CI status and other health metrics.
-- [website](https://github.com/ipfs/website) - The source to the IPFS community website at http://ipfs.io.
-
-#### Ref Lists
 
-- [refs](https://github.com/ipfs/refs) - Tools for publishing lists of IPFS ref heads.
-- [refs-denylists-dmca](https://github.com/ipfs/refs-denylists-dmca) - DMCA takedown notices for the IPFS Public Gateway.
-- [refs-solarnet-storage](https://github.com/ipfs/refs-solarnet-storage) - Inventory of content archived on Solarnet storage hosts.
+### Apps and data sets on IPFS
 
-### Other Community Resources
+- [Awesome IPFS](https://awesome.ipfs.io) - an ever-growing list of apps, data sets, and other inspirational resources built on IPFS.
 
-- [examples](https://github.com/ipfs/examples) - Examples on how to use go-ipfs.
-- [awesome-ipfs](https://github.com/ipfs/awesome-ipfs) - Useful resources for using IPFS and building things on top of it.
-- [logo](https://github.com/ipfs/logo) - The logo for IPFS.
+### Specs, papers, and discussion
 
-#### IPFS Meetups
-
-There are plethora of IPFS Meetups all over the world now. Find yours at the [Community Repo](https://github.com/ipfs/community#events)
-
-We encourage and support IPFS Meetups, please let us know if you would like to start one. Feel free to organize yourself through [the Community discussions](https://github.com/ipfs/community/issues) and [to advertise events in the main repository](https://github.com/ipfs/ipfs/issues).
-
-#### Tools
+- [specs](https://github.com/ipfs/specs) - Specifications on the IPFS protocol.
+- [papers](papers) - Academic papers on IPFS.
+- [IPFS Docs: Further Reading](https://docs-beta.ipfs.io/concepts/further-reading/academic-papers) - Papers to read to understand IPFS and its underlying technologies.
+- [notes](https://github.com/ipfs/notes) - The IPFS collaborative notebook for research, used for discussing known open IPFS problems, nascent issues, proposals, or features.
```

Comment:
Maybe we should leave /notes out and name the section `Specs and Papers` only. We need to start phasing out /notes, or at least making it a more specific/focused discussion place.

Reply:
Done.